hub/apps/how-tos/uno-multiplatform.md

```diff
@@ -45,75 +45,101 @@ Now that you've created your first WinUI 3 app, you might be wondering how to re
     :::image type="content" source="images/hello-world/vs2022-project-reload.png" alt-text="Visual Studio banner offering to reload your projects to complete changes":::
 
-## Copy your existing WinUI 3 project items
+## Building your app
+
+Now that you've generated the functional starting point of your multi-platform WinUI application, you can copy markup into it from the Hello World WinUI 3 project outlined in the [previous](/hub/apps/how-tos/hello-world-winui3.md) tutorial.
+
+1. Make sure Visual Studio has your WinUI 3 project open, then copy the child XAML elements from `MainWindow.xaml` in the WinUI 3 project to the your `MainPage.xaml` file in the Uno Platform project. The `MainPage` view markup should look like this:
 
-Now that you've generated the functional starting point of your multi-platform WinUI application, you can copy the existing WinUI 3 project items into it.
+```xml
+<Page x:Class="HelloWorld.MainPage"
+	  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	  xmlns:local="using:HelloWorld"
+	  xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	  xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	  mc:Ignorable="d"
+	  Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
 
-1. Make sure Visual Studio has your WinUI 3 project open, then copy the following items from the WinUI 3 project to the Uno Platform project:
-    - `MainWindow.xaml`
-    - `MainWindow.xaml.cs`
+      <!-- Below is the code you copied from MainWindow: -->
+      <StackPanel Orientation="Horizontal"
+                  HorizontalAlignment="Center"
+                  VerticalAlignment="Center">
+          <TextBlock x:Name="myText" 
+                     Text="Hello world!"
+                     Foreground="Red"/>
+      </StackPanel>
+</Page>
+```
 
-2. If you're prompted to do so, overwrite any existing files of the same name.
 
-## Deploy your application
 
-1. To run the **WebAssembly** (Wasm) head:
-    - Right click on the `MyApp.Wasm` project, select **Set as startup project**
+2. To run the **WebAssembly** (Wasm) head:
+    - Right-click on the `MyApp.Wasm` project, select **Set as startup project**
     - Press the `MyApp.Wasm` button to deploy the app
-2. To run the ASP.NET Hosted **WebAssembly** (Server) head:
-    - Right click on the `MyApp.Server` project, select **Set as startup project**
+3. To run the ASP.NET Hosted **WebAssembly** (Server) head:
+    - Right-click on the `MyApp.Server` project, select **Set as startup project**
     - Press the `MyApp.Server` button to deploy the app
-3. To debug for **iOS**:
-    - Right click on the `MyApp.Mobile` project, select **Set as startup project**
+4. To debug for **iOS**:
+    - Right-click on the `MyApp.Mobile` project, select **Set as startup project**
     - In the "Debug toolbar" drop-down, select framework `net7.0-ios`:
```

Comment:
Skip picking framework and just pick either active ios device or simulator. Note that if not paired with mac, not devices and simulators will show up eg
![image](https://user-images.githubusercontent.com/1614057/227870932-251d63b7-ed41-4eba-8d29-8e53f414c36c.png)